lightly/active_learning/agents/agent.py

```diff
@@ -76,28 +76,29 @@ def _set_labeled_and_unlabeled_set(self, preselected_tag_data: TagData = None):
                 optional param, then it must not be loaded from the API
 
         """
-        if self.preselected_tag_id is None:
-            self.labeled_set = []
-        else:
+        self.bitmask_labeled_set = BitMask.from_hex("0x0")
+        self.bitmask_added_set = BitMask.from_hex("0x0")
+        if self.preselected_tag_id is not None:
```

Comment:
Also a comment here please describing what happens if the preselected tag id is not None